app/support/route.ts

```diff
@@ -1,3 +1,16 @@
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
-export default class Support extends Route {}
+export default class Support extends Route {
+    analytics = service();
+    routah = service('router');
+    actions = {
+        didTransition(this: Support) {
+            const page = this.get('routah').currentUrl;
```

Comment:
In several other routes you use `this.get('router')` without the service injection, which means it's using the soon-to-be-deprecated internal router service. Could this use that too, and be changed when we upgrade to 3.2?

Reply:
I think in the other two places I did this, I used `router = service();` or the equivalent and didn't have to modify the name. I didn't have to, because yes, but @jamescdavis asked that we inject the service because it's deprecated. He was going to fix the underlying problem in another PR as of when this tiny hack happened.

Reply:
Orrrr I thought I did and that never happened. Huh.

Reply:
So, `Route.router` isn't actually deprecated until Ember 3.2. If we just continue to say `this.get('router')` it will still work in 3.2, we'll just have to start injecting the router service.

Reply:
Also, I haven't opened the PR yet, but I'm not having any issue using `@service router` in a route.